### PR TITLE
Pluralize

### PR DESCRIPTION
**Pull request by bgeuken**

### CODE REVIEW

**Comment by bgeuken** [comment]
Note: This causes a small change in the ui.

Before:
before

After:
after

Same goes for "open request"

File: src/api/app/views/webui/project/show.html.erb [hunk 1]
```diff
@@ -29,14 +29,14 @@
...
-              <%= @maintained_projects.length %> <%= link_to("maintained project#{@maintained_projects.length == 1 ? '' : 's'}", action: 'maintained_projects', project: @project) %>
+              <%= link_to(pluralize(@maintained_projects.count, "maintained project"), action: 'maintained_projects', project: @project) %>
             </li>
         <% else # also for incident project                                    %>
             <% if @nr_of_problem_packages and @nr_of_problem_packages > 0 %>
                 <li>
                   <%= sprite_tag 'exclamation' %>
                   <%= @nr_of_problem_packages %>
-                  <%= link_to "build error#{@nr_of_problem_packages == 1 ? '' : 's'}", action: 'monitor', project: @project, succeeded: 0, blocked: 0, finished: 0, signing: 0, dispatching: 0, scheduled: 0, building: 0 %>
+                  <%= link_to(@nr_of_problem_packages, "build error", action: 'monitor', project: @project, succeeded: 0, blocked: 0, finished: 0, signing: 0, dispatching: 0, scheduled: 0, building: 0) %>
```
Comment (ChrisBr) [inline src/api/app/views/webui/project/show.html.erb]: Isn't here the pluralize missing?

``
 link_to(pluralize(@nr_of_problem_packages, "build error"), action: 'monitor', project: @project, succeeded: 0, blocked: 0, finished: 0, signing: 0, dispatching: 0, scheduled: 0, building: 0) %>
``

**Comment by ChrisBr** [comment]
Except my comment it LGTM! :+1:

File: src/api/app/views/webui/project/show.html.erb (hunk 1 above)
Comment (bgeuken) [inline src/api/app/views/webui/project/show.html.erb]: True that

**Pull request merged**